spec/services/attribute_asserter_spec.rb

```diff
@@ -327,6 +327,31 @@
         end
       end
 
+      context 'service provider specifies attribute bundle' do
```

Comment:
**Question:** This is nested under the context `verified user and IAL1 request` - since we're testing IAL2 does it make sense to either move it to a different context or create a new one that makes sense?

Reply:
Thanks!